docs/breakpad.rst

```diff
@@ -0,0 +1,76 @@
+============================
+Breakpad and crash reporters
+============================
+
+Links about breakpad
+====================
+
+Breakpad project home page:
+    https://chromium.googlesource.com/breakpad/breakpad
+
+Firefox Breakpad page:
+    https://wiki.mozilla.org/Breakpad
+
+    Note: A lot of this is out of date.
+
+Socorro docs:
+    http://socorro.readthedocs.io/en/latest/
+
+    Notes on testing collector and processor:
+    http://socorro.readthedocs.io/en/latest/configuring-socorro.html#test-collection-and-processing
+
+
+Where do reports come from?
+===========================
```

Comment:
I can nix this whole section (both times) and replace it with a link to the crash reporter docs.